migration/transformation_schema_2_to_1.yaml

```diff
@@ -0,0 +1,134 @@
+        # Classes metrics
+        - rename_metrics:
+#            jvm.class.loaded:   process.runtime.jvm.classes.loaded
+            jvm.class.unloaded: runtime.jvm.classes.unloaded  # micrometer
+            jvm.class.count:    runtime.jvm.classes.loaded    # micrometer
+
+        #--------------------------------------------------------------
+        # CPU metrics
+#        - rename_metrics:
+#            jvm.cpu.time:               process.runtime.jvm.cpu.time
+#            jvm.cpu.recent_utilization: process.runtime.jvm.cpu.recent_utilization
+#            jvm.system.cpu.utilization: process.runtime.jvm.system.cpu.utilization # experimental
+#            jvm.system.cpu.load_1m:     process.runtime.jvm.system.cpu.load_1m     # experimental
+
+#       #--------------------------------------------------------------
+#       # Garbage collection metrics
+#        - rename_metrics:
+#            jvm.gc.duration: process.runtime.jvm.gc.duration # otel
+#        - rename_attributes:
+#            attribute_map:
+#              name: jvm.gc.name
+#              action: jvm.gc.action
+#            apply_to_metrics:
+#              - jvm.gc.duration
+
+        #--------------------------------------------------------------
+        # Buffer metrics
+        - rename_metrics:
+            jvm.buffer.memory.usage: runtime.jvm.buffer.memory.used    # micrometer, experimental
+            jvm.buffer.memory.limit: runtime.jvm.buffer.total.capacity # micrometer, experimental
+            jvm.buffer.count:        runtime.jvm.buffer.count          # micrometer, experimental
+        - rename_attributes:
+            attribute_map:
+              jvm.buffer.pool.name: pool
+            apply_to_metrics:
+              - jvm.buffer.memory.usage
+              - jvm.buffer.memory.limit
+              - jvm.buffer.count
+
+        #--------------------------------------------------------------
+        # DB pool metrics
+        - rename_metrics:
+            db.client.connections.usage:            db.pool.connections                 # micrometer
+            db.client.connections.idle.min:         db.pool.connections.idle.min        # micrometer
+            db.client.connections.idle.max:         db.pool.connections.idle.max        # micrometer
+            db.client.connections.max:              db.pool.connections.max             # micrometer
+            db.client.connections.pending_requests: db.pool.connections.pending_threads # micrometer
+            db.client.connections.timeouts:         db.pool.connections.timeouts        # micrometer
+#           db.client.connections.create_time:      db.pool.connections.create_time     # micrometer, timer vs. histogram
+#           db.client.connections.wait_time:        db.pool.connections.wait_time       # micrometer, timer vs. histogram
+#           db.client.connections.use_time:         db.pool.connections.use_time        # micrometer, timer vs. histogram
```

Comment:
Why are these commented out?

Reply:
They were gray in the spreadsheet and don't have semantic equivalents afaik...largely due to the way micrometer timers are handled.